File: ToDoList/Back-End/ToDoList.Data/Entities/ListType.cs
namespace ToDoList.Data.Entities;
public enum ListTypes{
    Private = 1,
    Shared,
    Administrative
}
public class ListType {
    public Guid Id { get; set; }
    public ListTypes Type { get; set; } 
    public ICollection<AssignmentList> AssignmentLists { get; set; }
}

File: ToDoList/Front-End/ToDoListFront/src/app/models/List-Type.ts
import { Guid } from "guid-typescript";
import { AssignmentList } from "./AssignmentList/assignment-list";

export enum ListType{
    Administrative = 'Administrative',
    Shared = 'Shared',
    Normal = 'Normal',
}

export class AssignmentListType{
    public id?: Guid;
    public listType?: ListType;
    public assignmentList?: AssignmentList[];
}
